forc-plugins/forc-crypto/src/keygen/parse_secret.rs

```diff
@@ -0,0 +1,59 @@
+//! This file is migrated from https://github.com/FuelLabs/fuel-core/blob/master/bin/keygen/src/keygen.rs
```

Comment:
Yes, it is already done https://github.com/FuelLabs/fuel-core/pull/1397, but I'm waiting for a fix to their Cargo.toml to be able to include the code directly. Until that happens it will be copied. I have a follow up ticket already https://github.com/FuelLabs/sway/pull/5176 5170

Reply:
In this situation I would either like to wait for the upstream changes to be available, or for an issue to be opened and a link to the issue be left in the comments so we don't forget to clean up.

Reply:
I will be happy with just including this URL in the comments for now https://github.com/FuelLabs/sway/issues/5170

Reply:
Done in 6c4545c18